app/react/src/server/types.ts

```diff
@@ -0,0 +1,10 @@
+import { StorybookOptions as BaseOptions } from '@storybook/core/types';
```

Comment:
should this be `import type`? does it matter?

Reply:
I think it doesn't matter much in this case. It's just a new syntax for the same thing. It's nice when there are mixed imports (types and non-types) because it makes the code more understandable, but in that case I personally feel it doesn't make a difference